libs/aws/langchain_aws/llms/bedrock.py

```diff
@@ -1216,13 +1234,16 @@ def _prepare_input_and_invoke_stream(
 
         body = json.dumps(input_body)
 
-        request_options = {
+        request_options: dict[str, Any] = {
             "body": body,
             "modelId": self.model_id,
             "accept": "application/json",
             "contentType": "application/json",
         }
 
+        if self.service_tier:
+            request_options["serviceTier"] = {"type": self.service_tier}
+
```

Comment:
Ditto as for `invoke_model`, [`invoke_model_with_response_stream`](invoke_model_with_response_stream) also expects tier as str input:
```suggestion
            request_options["serviceTier"] = self.service_tier
```